src/main/java/reposense/git/GitBlame.java

```diff
@@ -57,6 +63,23 @@ public static String blameWithPreviousAuthors(String root, String fileDirectory)
         return StringsUtil.filterText(runCommand(rootPath, blameCommandWithFindingPreviousAuthors), COMBINATION_REGEX);
     }
 
+    /**
+     * Returns the processed git blame result, created from the raw git blame result.
+     */
+    public static List<GitBlameLineInfo> blameFile(String blameResults) {
```

Comment:
Could we have the interface to `blameFile` be similar to that of `blameLine` where the blame is done internally? We could have `blameFile` require `root`,  `fileDirectory` and a flag for `withPreviousAuthors` for now. We could then probably get rid of `getGitBlameResult` in `FileInfoAnalzyer`.